plugins/osx/osx.plugin.zsh

```diff
@@ -161,6 +161,17 @@ function itunes() {
 		vol)
 			opt="set sound volume to $1" #$1 Due to the shift
 			;;
+		playing|status)
+			local state=`osascript -e 'tell application "iTunes" to player state as string'`
+			if [[ "$state" = "playing" ]]; then
+				currenttrack=`osascript -e 'tell application "iTunes" to name of current track as string'`
+				currentartist=`osascript -e 'tell application "iTunes" to artist of current track as string'`
+				echo -E "Listening to $fg[yellow]$currenttrack$reset_color by $fg[yellow]$currentartist";
```

Comment:
Should close this with another `$reset_color` to avoid contaminating subsequent output that happens before a prompt or other colorizing output resets it.

<img width="699" alt="screen shot 2015-09-20 at 6 03 01 pm" src="https://cloud.githubusercontent.com/assets/2618447/9983186/fa5a309a-5fc1-11e5-8984-888388cbf208.png">